components/buttons/Button.tsx

```diff
@@ -1,15 +1,31 @@
 import Link from 'next/link';
 import { twMerge } from 'tailwind-merge';
 
+export enum ButtonSizeType {
+  small = 'small',
+  default = 'default',
+}
```

Comment:
Its name should be `ButtonSize` as it is not representing a type but is a size variant instead, which is self-explanatory.

Reply:
Done